test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
```

Comment:
On purpose, the excludes should live in the config file. As mentioned in the PR body.